trio/_highlevel_open_tcp_listeners.py

```diff
@@ -240,6 +237,7 @@ async def serve_tcp(
 
     """
     listeners = await trio.open_tcp_listeners(port, host=host, backlog=backlog)
+    # typecheck: no-untyped-call error: Call to untyped function "serve_listeners" in typed context
```

Comment:
nit, but I don't think we want to include action comments that refers to checkers or options not used in CI. Users should generally handle those themselves, or request them be added to CI, as there's no checking that they're kept updated, removed if no longer used, etc.

Reply:
Yea that comment is not supposed to be there, that's a remnant of using my mypy extension in the text editor, I'll remove it.

Reply:
Removed in 6ff13c4